roles/openshift_register_nodes/library/kubernetes_register_node.py

```diff
@@ -369,7 +396,7 @@ def main():
             module.fail_json(msg="Unknown error creating node",
                              node=node.get_node())
 
-
+# pylint: disable=redefined-builtin, unused-wildcard-import, wildcard-import
```

Comment:
same as above. I don't think we should allow pylint disables without a comment explaining why.